gradle/libs.versions.toml

```diff
@@ -77,8 +77,8 @@ opencensus-exporter-stats-stackdriver = { module = "io.opencensus:opencensus-exp
 opencensus-exporter-trace-stackdriver = { module = "io.opencensus:opencensus-exporter-trace-stackdriver", version.ref = "opencensus" }
 opencensus-impl = { module = "io.opencensus:opencensus-impl", version.ref = "opencensus" }
 opencensus-proto = "io.opencensus:opencensus-proto:0.2.0"
-opentelemetry-api = "io.opentelemetry:opentelemetry-api:1.32.0"
-opentelemetry-sdk-testing = "io.opentelemetry:opentelemetry-sdk-testing:1.32.0"
+opentelemetry-api = "io.opentelemetry:opentelemetry-api:1.36.0"
```

Comment:
Is there a reason why we are upgrading to `1.36` instead of the latest release i.e `1.37`?

Reply:
I'm currently using 1.34 of opentelemetry-java-contrib (as 1.35 was recently released) and contrib 1.34 uses otel 1.36. I could push the version further, but it is easier to get everything ready on the slightly older version and then jump to latest.